packages/workspace/src/core/file-utils.ts

```diff
@@ -172,11 +172,7 @@ export function cliCommand() {
 }
 
 export function workspaceFileName() {
-  const packageJson = readPackageJson();
-  if (
-    packageJson.devDependencies['@angular/cli'] ||
-    packageJson.dependencies['@angular/cli']
-  ) {
+  if (fileExists('angular.json')) {
```

Comment:
Use `appRootPath` to make sure this works when executing from somewhere other than the root of a workspace.
```suggestion
  if (fileExists(`${appRootPath}/package.json`)) {
```